docs/content/docs/test/Runner.md

```diff
@@ -672,16 +672,16 @@ instrumentation {
         local {
             tryToReRunOnTargetBranch = false
             targets {
-                api27 {
-                    deviceName = "API27"
+                api28 {
+                    deviceName = "API28"
                     scheduling {
                         quota {
                             retryCount = 1
                             minimumSuccessCount = 1
                         }
 
                         testsCountBasedReservation {
-                            device = new LocalEmulator("27", 27, "Android_SDK_built_for_x86")
+                            device = new LocalEmulator("28", 28, "Android_SDK_built_for_x86_64")
```

Comment:
I think we should reconsider how DeviceWorker selects available devices.
As a user, I want to request only an API level. I have different local devices with different device names.

Reply:
I think we must add api like a
emulator27 {
}
emulator22 {
}
and
customEmulator {
imagePath =
version = 
}

Reply:
I agree that we should reconsider DeviceWorker's mechanism but I believe it's out of the scope of this task.